### ISSUE TITLE
better highlight link targets in "learn more" sections

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-quick-start#39]
Suggest using the link name for recognition of the target, *in addition* to a prefix to better indicate what the link is (especially for skimming). For example:
<strong>WCAG Requirement:</strong> <a=quickref>X.Y.Z [shortname]</a>
<strong>Background:</strong> <a=understanding>Understanding X.Y.Z</a>
<strong>User Story:</strong> <a=how-people-use-web>Mrs/Mr ABC [story title]</a>

This may be more relevant to the "designing", "authoring", and "developing" pages. The "managing" and "advocacy" pages may have something like:
<strong>How To:</strong> <a=business-case>Developing a Customized Business Case</a>
<strong>Background</strong> <a=introduction>Introduction to Web Accessibility</a>

It may be difficult to come up with a consistent set of labels but I think it is important for readability. Also, being driven by "how to" and "background" resources (where available), may help sharpen the "actionability" of the tips themselves. We should avoid having too many tips without some form of "how to" and "background" resources (the "how to" is the same as "quickref" for some pages).

**Comment by iadawn** [comment]
Three approaches to learn more links are presented:

[Shorter more action focused link text with no extra description](http://w3c.github.io/wai-quick-start/designing.html#provide-sufficient-color-contrast)
[As per above @nitedog suggestion above](http://w3c.github.io/wai-quick-start/designing.html#dont-use-color-alone-to-signify-meaning)
[Original style](http://w3c.github.io/wai-quick-start/designing.html#ensure-interactive-elements-are-easy-to-identify)

**Comment by hkramer** [comment]
Viewing the current version, which seems to incorporate what is stipulated above, works well for quick scanning and recognition of the learn more links.

**Comment by slhenry** [comment]
I strongly support Shadi's suggestion (#2 of the 3 approaches). I think it helps with skimming and more clearly indicates what the link it.
I also don't think we need to worry too much about consistency. Yes, it's good where useful; however, I think we should use whatever is easiest to understand for each case, and it's OK if there is some inconsistency.
I also think that we can include additional information where it's provides more clarity than clutter -- e.g., some of the wording in #1.

**Comment by bakkenb** [comment]
I like the first and second approaches. Use the second when a consistent title can be used over and over again, then revert back to the first - shorter actionable link with no extra description when there is not a consistent title to be applied.

**Comment by schantzwebconsulting** [comment]
I prefer the second approach, and think we should be consistent.

**Comment by AndrewArch** [comment]
Like Brent, I have a preference for #2 when it works.

**Comment by Egyrs** [comment]
Idem, +1 to #2 generally.

**Comment by iadawn** [comment]
[EOWG Meeting 26 Jun](http://www.w3.org/2015/06/26-eo-minutes.html#item05): Resolved to style links as in option 2.
